packages/cli/src/services/workflow-statistics.service.ts

```diff
@@ -10,6 +16,35 @@ import { TypedEmitter } from '@/typed-emitter';
 
 import { OwnershipService } from './ownership.service';
 
+const isStatusBillable = {
+	success: true,
+	crashed: true,
+	error: true,
+
+	canceled: false,
+	new: false,
+	running: false,
+	unknown: false,
+	waiting: false,
+} satisfies Record<ExecutionStatus, boolean>;
+
+const isModeBillable = {
```

Comment:
Do we want to remove `billable` here?

```suggestion
const isModeRootExecution = {
```